scripts/esb_sniffer/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _esb_sniffer_scripts_docs:
+
+ESB Sniffer
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+This directory contains python scripts for the :ref:`esb_monitor` sample configured as ESB Sniffer.
+
+Setup Linux
+***********
+
```

Comment:
Would be good to have a short introductory sentence before jumping straight to the steps. For example: Complete the following steps to set up your Linux environment for the scripts: